back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -148,7 +148,6 @@ def update_field!(field, field_params, errors, index)
     end
 
     def delete_field!(field)
-      SideFxCustomFieldService.new.before_destroy field, current_user
```

Comment:
A main bugfix (of 2).

Since `before_destroy` only calls `UserCustomFieldService.new.delete_custom_field_option_values`, which is only concerned with removing `User.custom_field_values` (either just the values, or the key also if the value is the only value for the key), we now don't call it.